src/Mod/Arch/importIFC.py

```diff
@@ -1538,6 +1544,8 @@ def export(exportList,filename,colors=None):
     context = ifcfile.by_type("IfcGeometricRepresentationContext")[0]
     project = ifcfile.by_type("IfcProject")[0]
     objectslist = Draft.getGroupContents(exportList,walls=True,addgroups=True)
+    trueNorthX = math.tan(-Draft.getObjectsOfType(objectslist, "Site")[0].Declination.getValueAs(FreeCAD.Units.Radian)) # we assume one site and one representation context only
```

Comment:
This will fail if the model contains no site, which is a possibility. Better to split in two lines, with something like: `if Draft.getObjectsOfType(objectslist, "Site"): ...`

Reply:
Indeed, added the check in 0fca16f

Reply:
Great, thanks!